mdp/nodes/stats_nodes_online.py

```diff
@@ -0,0 +1,93 @@
+import mdp
+
+
+class MovingAvgNode(mdp.PreserveDimOnlineNode):
```

Comment:
There are two parts to this comment:

First on the definition of the moving average:
MovingAvgNode supports two averages: (a) A cumulative moving average according to the wiki page you have pointed out, which is also an online average calculation. (b) An exponentially weighted moving average. All moving averages more or less follow an update equation similar to:
avg_t = alpha * x_t + (1-alpha) * avg_t-1. For cumulative moving avg: alpha = 1/t and for exponential it is a constant. So I believe in this regard the usage of moving average as its name is justified. 

 However, after taking a second look, I figured out that this may not be a right name for the node after all. Since the node does not return an average but returns the centered data. So, I aptly changed the name of the node to OnlineCenteringNode and improved the docstring. The node's main purpose is to center the data via an execute call by updating an average via the train call. 

Thanks for pointing this out anyway :)